frontend/public/components/monitoring/receiver-forms/alert-manager-receiver-forms.tsx

```diff
@@ -358,10 +390,73 @@ const ReceiverBaseForm: React.FC<ReceiverBaseFormProps> = ({
   );
 };
 
+export const SaveAsDefaultCheckbox: React.FC<SaveAsDefaultCheckboxProps> = ({
+  formField,
+  disabled,
+  label,
+  formValues,
+  dispatchFormChange,
+  tooltip,
+}) => {
+  const saveAsDefaultLabelClass = classNames('checkbox', { 'co-no-bold': disabled });
+  return (
+    <label className={saveAsDefaultLabelClass}>
+      <input
+        type="checkbox"
+        name={formField}
+        data-test-id="save-as-default"
+        onChange={(e) =>
+          dispatchFormChange({
+            type: 'setFormValues',
+            payload: { [formField]: e.target.checked },
+          })
+        }
+        checked={formValues[formField]}
+        aria-checked={formValues[formField]}
+        aria-label={tooltip}
```

Comment:
I don't think this is right... You want `aria-describedby`. This will break screenreaders using the label. I would leave this off.

Reply:
fixed